modules/apps/product-navigation/product-navigation-applications-menu-web/src/main/java/com/liferay/product/navigation/applications/menu/web/internal/portlet/action/ApplicationsMenuLiferayLogoMVCResourceCommand.java

```diff
@@ -68,8 +68,9 @@ protected void doServeResource(
 		if (inputStream == null) {
 			return;
 		}
-
-		PortletResponseUtil.write(resourceResponse, inputStream);
+		PortletResponseUtil.sendFile(
```

Comment:
Hi @chileces 
Could you please try the below solution 
```
resourceResponse.setContentType(ContentTypes.IMAGE_PNG);
PortletResponseUtil.write(resourceResponse, inputStream);
```
Thank you.

Reply:
@hongvo2308 thank you for your suggestion, i did it